registry.yaml

```diff
@@ -9401,6 +9401,27 @@ packages:
       - darwin
       - linux/amd64
     asset: gossh-{{.Version}}-{{.OS}}-{{.Arch}}
+  - type: github_release
+    repo_owner: wtetsu
+    repo_name: gaze
+    asset: gaze_{{.OS}}_{{.Version}}.{{.Format}}
+    format: zip
+    description: Executes commands for you
+    replacements:
+      darwin: macos
+    overrides:
+      - goos: darwin
+        asset: gaze_{{.OS}}_amd_{{.Version}}.{{.Format}}
+        files:
+          - name: gaze
+            src: gaze_{{.OS}}_amd_{{.Version}}/gaze
+    supported_envs:
+      - darwin
+      - amd64
+    rosetta2: true
```

Comment:
https://github.com/wtetsu/gaze/blob/06b7a8ebfd7409c69b57a6df6c1b2e549871e27c/Makefile#L12-L13

There is an asset for darwin/arm64, so `rosetta2` isn't needed.